### DISCUSSION TITLE
Медали для бено

### DISCUSSION THREAD

**Discussion by Shootnik01** [Cosmic-Overlord/TerraGov-Marine-Corps#312] [Ideas]
Добавить аналог медалей марин для бно. Вроде на Кумах было шо-то такое, феромоны награды, или как-то так..

**Comment by fe-ed** [comment]
Можно что-то придумать интересное, чтоб у награжденного ксена был какой-нибудь визуальный прикол, а то и вовсе механ

**Reply by Shootnik01** [reply]
Ну, во-первых, значок особого феромона, про механ хз, медали тогда будут просто так давать, чтоб хайв сильнее был. Но что-нибудь в чат, типа you feel a thin scent of blueberry, когда он рядом, плюс, внешне ксеноморф немного переливается цветом медали (ну, в зависимости от вида медали)
